src/lib/views/announcementPage/announcement_details.dart

```diff
@@ -0,0 +1,236 @@
+                                  "Show contact details",
+                                  style: TextStyle(
+                                    color: Colors.white,
+                                  ),
+                                ),
+                                const SizedBox(height: 5),
+                                SizedBox(
+                                  width: 100,
+                                  child: Row(
+                                    mainAxisAlignment:
+                                        MainAxisAlignment.spaceEvenly,
+                                    children: const [
+                                      Icon(
+                                        Icons.phone,
+                                        color: Colors.white,
+                                      ),
+                                      SizedBox(
+                                        width: 5,
+                                      ),
+                                      Icon(
+                                        Icons.mail,
+                                        color: Colors.white,
+                                      ),
+                                    ],
+                                  ),
+                                ),
+                              ],
+                            ),
+                            secondChild: SizedBox(
+                              width: 300,
+                              child: Row(
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceEvenly,
+                                children: [
+                                  Column(
+                                    children: const [
+                                      Text(
+                                        "Phone number:",
+                                        style: TextStyle(
+                                          color: Colors.white,
+                                        ),
+                                      ),
+                                      SizedBox(
+                                        height: 5,
+                                      ),
+                                      Text(
+                                        "E-mail address:",
+                                        style: TextStyle(
+                                          color: Colors.white,
+                                        ),
+                                      )
+                                    ],
+                                  ),
+                                  const SizedBox(width: 5),
+                                  Column(
+                                    children: [
+                                      const Text(
+                                        // widget.announcement.author.phoneNumer, // problem, nullowalne
+                                        "999 999 999",
```

Comment:
Póki co zahardcodowane, nie miałam czasu implementować na szybko numeru telefonu.

Reply:
No problem, i know how to fix it :D